queries/ralph/highlights.scm

```diff
@@ -0,0 +1,91 @@
+(type_identifier) @type
+(primitive_type) @type.builtin
+(alph_token_id) @type.builtin
+
+(var_declaration) @variable
+(struct_field) @variable
+(arg_def (identifier) @variable)
+(struct_constructor_field (identifier) @variable)
+
+(hex_num) @number
+(integer) @number
+(typed_num) @number
```

Comment:
Would you mind grouping the captures to make things easier? E.g. for this one it would be:
```query
[
  (hex_num)
  (integer)
  (typed_num)
] @number
```
Also, please run this file through the formatter found at `scripts/format-queries.lua`